buildflow/runtime/managers/stream_manager.py

```diff
@@ -206,17 +207,26 @@ async def run(self):
                         continue
                     events_processed.append(metric.num_events)
                     non_empty_ratios.append(metric.non_empty_response_ratio)
+
+                total_events_process = sum(events_processed)
+                if total_events_process > 0:
+                    self.num_events_counter.inc(total_events_process)
```

Comment:
random idea: Would be cool if we tracked this globally, like maybe check in with the buildflow usage endpoint. Then we could eventually say LaunchFlow has "processed X billion events" or something

Reply:
Oh yea that would be cool, I think we could do something like that once we have historical metrics.